common/technologies/infantry.txt

```diff
@@ -163,7 +163,7 @@ technologies = {
 		}
 
 		ai_will_do = {
-			factor = 1000000
+			factor = 1000

```

Comment:
Why? Has this been balance approved to see its effect?

Reply:
Too big value turned into negative when multiplied by AI factors. I have no idea who might need it, but I will give it back if necessary.